lib/libc/aarch64/gen/_setjmp.S

```diff
@@ -37,13 +37,6 @@ ENTRY(_setjmp)
 	mov	REG(9), REGN(sp)
 	stp	REG(8), REG(9), [REG(0)], #(REG_WIDTH * 2)
 
-#if defined(__CHERI_PURE_CAPABILITY__) && defined(RTLD_SANDBOX)
```

Comment:
Why are these setjmp diffs in the benchmark ABI commit?

Reply:
I reworked the implementation of _rtld_setjmp/_rtld_longjmp while porting to the benchmark ABI.

Reply:
Well don't? You have the big commit that adjusts them the first time, so put the final form there, don't change them once then change them again in a later commit that has nothing to do with setjmp/longjmp.